azurelinuxagent/common/protocol/wire.py

```diff
@@ -692,6 +715,16 @@ def update_certs(self, goal_state):
         self.save_cache(local_file, xml_text)
         self.certs = Certificates(self, xml_text)
 
+    def update_remote_access_conf(self, goal_state):
```

Comment:
There is a clear pattern for updating documents which are part of the goal state, and this does not fit. See [update_goal_state](https://github.com/Azure/WALinuxAgent/blob/master/azurelinuxagent/common/protocol/wire.py#L707)

Reply:
Updated